### DISCUSSION TITLE
App.json file inacccesible on Larvel App

### DISCUSSION THREAD

**Discussion by Samostic242** [dokku/dokku#5852] [Q&A]
Hello please i am trying to setup a cron task on my Laravel application on Dokku, but can't access the app.json file to add the cron task

**Comment by josegonzalez** [comment]
@Samostic242 can you provide more detail about your app? Since its your repository codebase, I don't see why you wouldn't have access to that file.

**Comment by tkemi** [comment]
@Samostic242 I have managed to set up cron on my laravel app like this:

- Add app.json file in the root of your project(meaning where Procfile, artisan, composer... are located)

- In the app.json file add something like this:
```
{
    "cron": [
        {
            "command": "php artisan kyc:status",
            "schedule": "*/10 * * * *"
        }
    ]
}
```

- Deploy it and you're good to go
